src/dreyfus/src/clouseau_rpc.erl

```diff
@@ -285,6 +287,14 @@ analyze(Analyzer, Text) ->
 version() ->
     rpc({main, clouseau()}, version).
 
+-spec clouseau_major_vsn() -> null | binary() | no_return().
+clouseau_major_vsn() ->
+    case version() of
+        {ok, <<Major, _/binary>>} -> <<Major>>;
+        {ok, null} -> null;
```

Comment:
Is there a case where the response is `null`?

Reply:
Thanks to Jay for reminding me of this.
If we run clouseau in dev mode, it will return `null`.
```bash
cd clouseau
mvn scala:run -Dname=clouseau1 -Dlauncher=with-cookie -Dcookie=<cookie>
```

Reply:
> If we run clouseau in dev mode, it will return null.

Please add a comment about it.

Reply:
Sorry, what "dev mode" are we talking about here?

Reply:
> Sorry, what "dev mode" are we talking about here?

When it's run via e.g. `make clouseau1`, and not from a jar file as it does in production.

Reply:
> > If we run clouseau in dev mode, it will return null.
> 
> Please add a comment about it.

Or change return value from `null` to something self explanatory.

```
 {ok, null} -> dev_mode;
 {ok, null} -> edge;
 {ok, null} -> dev;
 {ok, null} -> development;
 {ok, null} -> experimental;
```

Reply:
Albeit I acknowledge this is going to stay with us, so the `null` case should be supported somehow.  For that, I like @iilyak's idea.